### PR TITLE
Pass client ip to lettera as header

### PR DESCRIPTION
**Pull request by mrskug**

### CODE REVIEW

File: apps/mosaico/src/Main.purs [hunk 1]
```diff
@@ -247,19 +262,19 @@ getDraftArticle env { params: {aptomaId}, query } = do
...
+  -> { params :: { uuidOrSlug :: String }, guards :: { credentials :: Maybe UserAuth, clientip :: Maybe String } }
   -> Aff (Response ResponseBody)
-getArticle env r@{ params: { uuidOrSlug }, guards: { credentials } }
+getArticle env r@{ params: { uuidOrSlug }, guards: { credentials, clientip } }
   | Just uuid <- UUID.parseUUID uuidOrSlug = do
       { user, article, mostReadArticles, latestArticles } <- sequential $
         { user: _, article: _, mostReadArticles: _, latestArticles: _ }
         <$> maybe (pure Nothing) (parallel <<< getUser) credentials
-        <*> parallel (Lettera.getArticle uuid mosaicoPaper r.guards.credentials)
+        <*> parallel (Lettera.getArticle uuid mosaicoPaper r.guards.credentials clientip)
```
Comment (kaol) [inline apps/mosaico/src/Main.purs:272]: Let's just use `credentials clientip`. Both variables are in scope and it'd be odd style to to access one via `r` and the other directly.

File: apps/mosaico/src/Main.purs [hunk 2]
```diff
@@ -247,19 +262,19 @@ getDraftArticle env { params: {aptomaId}, query } = do
...
         <*> parallel (Cache.getContent <$> Cache.getMostRead env.cache)
         <*> parallel (Cache.getContent <$> Cache.getLatest env.cache)
       renderArticle env user article mostReadArticles latestArticles
   | otherwise = do
-    article <- Lettera.getArticleWithSlug uuidOrSlug r.guards.credentials
+    article <- Lettera.getArticleWithSlug uuidOrSlug r.guards.credentials clientip
```
Comment (kaol) [inline apps/mosaico/src/Main.purs:277]: Ditto.

**Review by kaol — APPROVED** [review]
Just a couple of style things.

File: apps/mosaico/src/Main.purs (hunk 1 above)
Comment (mrskug) [reply apps/mosaico/src/Main.purs:272]: I was wondering about that

**Pull request merged**